tests/vklayertests_buffer_image_memory_sampler.cpp

```diff
@@ -10128,8 +10128,13 @@ TEST_F(VkLayerTest, ExtensionNotEnabled) {
     ycbcr_create_info.chromaFilter = VK_FILTER_NEAREST;
     ycbcr_create_info.forceExplicitReconstruction = false;
     VkSamplerYcbcrConversion conversion;
-    vkCreateSamplerYcbcrConversionKHR(m_device->handle(), &ycbcr_create_info, nullptr, &conversion);
-    m_errorMonitor->VerifyFound();
+    if (vkCreateSamplerYcbcrConversionKHR) {
+        vkCreateSamplerYcbcrConversionKHR(m_device->handle(), &ycbcr_create_info, nullptr, &conversion);
+        m_errorMonitor->VerifyFound();
+    } else {
+        // This shouldn't happen because the above checks should cause an early skip if this extension isn't there.
+        printf("Unexpected nullptr for PFN_vkCreateSamplerYcbcrConversionKHR\n");
+    }
```

Comment:
`VkLayerTest::GetDeviceProcAddr` asserts on `vk::GetDeviceProcAddr` returning nullptr.

```suggestion
    auto vkCreateSamplerYcbcrConversionKHR =
GetDeviceProcAddr<PFN_vkCreateSamplerYcbcrConversionKHR>("vkCreateSamplerYcbcrConversionKHR");
    vkCreateSamplerYcbcrConversionKHR(m_device->handle(), &ycbcr_create_info, nullptr, &conversion);
    m_errorMonitor->VerifyFound();
```

Reply:
`assert` in only defined in debug builds I think